src/main/java/ui/RepositorySelector.java

```diff
@@ -49,6 +52,17 @@ private void setupComboBox() {
                 onValueChangeCallback.accept(repoId);
             }
         });
+        
+        // remove focus from the repo selector
+        comboBox.addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+            event.consume();
+            if (event.getCode() == KeyboardShortcuts.REMOVE_FOCUS) {
+                if (ui.isTestMode()) {
```

Comment:
if we're able to check for focus of the repo selector before and then no focus after the key has been pressed, i think that's a better way to do it. we can create a separate issue to change the existing tests.